<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSEnaMainWindow</class>
 <widget class="QMainWindow" name="QtSEnaMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="numbersGridBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="title">
     <string>Sistema</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="condizionamentoBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>200</width>
      <height>320</height>
     </rect>
    </property>
    <property name="title">
     <string>Condizionamento</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="minSumLayout">
      <item>
       <widget class="QLabel" name="minSumLabel">
        <property name="text">
         <string>Somma minima</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minSum">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>21</number>
        </property>
        <property name="maximum">
         <number>525</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>183</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="maxSumLayout">
      <item>
       <widget class="QLabel" name="maxSumLabel">
        <property name="text">
         <string>Somma massima</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxSum">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>21</number>
        </property>
        <property name="maximum">
         <number>525</number>
        </property>
        <property name="value">
         <number>525</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
